doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -242,6 +242,9 @@ nRF Desktop
 
 * Updated:
 
+  * The nRF desktop configurations that enable :ref:`nrf_desktop_bluetooth_guide_fast_pair` use MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) instead of ``B0`` bootloader.
+    This is done to support firmware updates using both :ref:`nrf_desktop_dfu` and :ref:`nrf_desktop_dfu_mcumgr`.
+  * The :ref:`nrf_desktop_dfu_mcumgr` is used instead of :ref:`nrf_desktop_smp` in MCUboot SMP configuration (:file:`prj_mcuboot_smp.conf`) for the nRF52840 DK target.
```

Comment:
```suggestion
  * The :ref:`nrf_desktop_dfu_mcumgr` is used instead of :ref:`nrf_desktop_smp` in MCUboot SMP configuration (:file:`prj_mcuboot_smp.conf`) for the nRF52840 DK build targets.
```
nRF52840 DK build targets Or nRF52840 DK?

Reply:
`nRF52840 DK`